File: galgamecommon/src/main/java/com/goexp/galgame/common/website/_2DFURL.java
package com.goexp.galgame.common.website;

import com.goexp.common.util.UrlBuilder;

public final class _2DFURL {

    public static final String searchUrl = "https://www.2dfan.com/subjects/search";

    public static String fromTitle(String title) {

        return UrlBuilder
                .create(searchUrl)
                .param("keyword", title)
                .build();
    }

}

File: galgamecommon/src/main/scala/com/goexp/galgame/common/website/_2DFURL.scala
package com.goexp.galgame.common.website

import com.goexp.common.util.UrlBuilder

object _2DFURL {
  private val searchUrl = "https://www.2dfan.com/subjects/search"

  def fromTitle(title: String): String = UrlBuilder.create(searchUrl).param("keyword", title).build
}